org-vet/org-vet.schema.json

```diff
@@ -0,0 +1,130 @@
+{
+  "$id": "EK0QuXKzVQ0yXaZYXRp_nuwDoxZpuH03_AP6qlZ-MV5m",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "Org Vet",
+  "description": "Authenticate an org with an explicit level of assurance.",
+  "type": "object",
+  "credentialType": "orgVet",
+  "version": "1.0.0",
+  "required": [
+    "v",
+    "d",
+    "i",
+    "ri",
+    "s",
+    "a",
+    "r"
+  ],
+  "properties": {
+    "v": {
+      "description": "Version string using ACDC conventions",
+      "type": "string"
+    },
+    "d": {
+      "description": "SAID of the credential",
+      "type": "string"
+    },
+    "i": {
+      "description": "Identifier of the issuer",
+      "type": "string"
+    },
+    "ri": {
+      "description": "Issuer's ACDC status registry",
+      "type": "string"
+    },
+    "s": {
+      "description": "SAID of this schema",
+      "type": "string"
+    },
+    "a": {
+      "$id": "EOt6UyUDfF4Ocw1UHAI8rhEQIHOTswqNS2sT2zaXreI6",
+      "description": "Attributes block",
+      "type": "object",
+      "required": [
+        "d",
+        "dt"
+      ],
+      "properties": {
+        "d": {
+          "description": "SAID of attributes block",
+          "type": "string"
+        },
+        "dt": {
+          "description": "Issuance date (when the ACDC was signed).",
+          "format": "date-time",
+          "type": "string"
+        },
+        "loa": {
+          "description": "Level of assurance.",
+          "type": "number"
+        },
+        "des": {
+          "description": "A human-friendly description of the content.",
+          "type": "string",
+          "examples": [
+            "Photo of the left front bumper of vehicle 1 from collision: a 2018 Ford F-150 XLT"
+          ]
+        },
+        "typ": {
+          "description": "The IANA content type of the cited content. This may help a verifier decide whether and how to download and process content.",
+          "type": "string",
+          "examples": [
+            "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+            "image/jpeg"
+          ]
+        },
+        "lbl": {
+          "description": "A convenient label or recommended filename for the content. This may help a verifier match the cited content to its metadata in an original context, but has no defined meaning with respect to how the ACDC is verified.",
+          "type": "string",
+          "examples": [
+            "invoice-03-25-2022.docx",
+            "Post on X on 25 July 2025"
```

Comment:
Can this be used to add actual LEI string here?

Reply:
this isn't the schema I thought was in my PR. I've made a mistake somewhere. Let me fix it.

Reply:
fixed